devtools/x-lint/src/lib.rs

```diff
@@ -190,6 +188,7 @@ pub mod prelude {
         file_path::{FilePathContext, FilePathLinter},
         package::{PackageContext, PackageLinter},
         project::{ProjectContext, ProjectLinter},
+        runner::{LintEngine, LintEngineConfig, LintResults},
```

Comment:
what's the impact of this line, what was happening when it wasn't here?   Trying to wrap my head around this.

Reply:
Previously we had made runner private and re-exported all its types to the top level. With this change runner is now public and its types are exported in the prelude. It's a minor change but I think it makes it better.